packages/flame/lib/src/components/joystick/joystick_action.dart

```diff
@@ -17,6 +17,9 @@ import 'joystick_events.dart';
 enum JoystickActionAlign { topLeft, bottomLeft, topRight, bottomRight }
 
 class JoystickAction extends BaseComponent with Draggable, HasGameRef {
+  @override
+  bool isHud = true;
```

Comment:
since these are children inside JoystickController, I wonder if this does anything at all.

Reply:
They should affect whether the child gets the widget or game position in the events, I thought this was the problem with the camera and the joystick together, but this didn't fix it.